### ISSUE TITLE
Documentation: Add new page, Image Library, as child of Using Backdrop

### ISSUE THREAD

**Issue by izmeez** [backdrop-ops/docs.backdropcms.org#238]
It may be helpful for new users and others to learn about the image library. How to access it and what it can do.

Searching the documentation fails to find an existing page on the "Image library". 

Even though image library is in core, there is no default admin bar menu item to content image library. Finding it can be a bit challenging.

**Comment by stpaultim** [comment]
Yes, the image library is really just an option for uploading images on some fields. It's not a thing on it's own. 

Some documentation could be useful. It could also be used better in core.

- https://github.com/backdrop/backdrop-issues/issues/6512
- https://github.com/backdrop/backdrop-issues/issues/1307
